lib/engine/game/g_18_co.rb

```diff
@@ -357,6 +403,12 @@ def event_remove_mines!
         end
       end
 
+      def event_unreserve_home_stations!
+        @log << '-- Event: Home station reservations removed --'
+
+        hexes.each { |h| h.tile.cities.map(&:remove_all_reservations!) }
```

Comment:
you did'nt resolve this this shouldn't be an each

Reply:
Argh, I changed the one but not the other.

Reply:
They should _both_ be each, shouldn't they?